compaction/time_window_compaction_strategy.hh

```diff
@@ -139,7 +139,7 @@ private:
     compaction_mode(const bucket_t& bucket, timestamp_type bucket_key, timestamp_type now, size_t min_threshold) const;
 
     std::vector<shared_sstable>
-    get_next_non_expired_sstables(table_state& table_s, std::vector<shared_sstable> non_expiring_sstables, gc_clock::time_point gc_before);
+    get_next_non_expired_sstables(table_state& table_s, std::vector<shared_sstable> non_expiring_sstables, gc_clock::time_point query_time);
```

Comment:
I think we should rename query_time to now, as compaction strategies will be only interacting with the interface which is agnostic to impl details.

Reply:
compaction_time is even better :-)

Reply:
Thanks. Fixed.